.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!-- To report an issue, fill in all the applicable fields below. -->
+
+#### Platform
+<!-- OS version, library version -->
+
+#### Steps to Reproduce
+1. 
+2.
+
+#### Expected result
+
+#### Observed result
+
+#### Possible cause
+
+#### Related issues
+
```

Comment:
Feels a bit too verbose for me, this issue seems to assume this is for just a "bug" type issue?